tests/cases/arithmetic_decimal/power.test

```diff
@@ -0,0 +1,21 @@
+### SUBSTRAIT_SCALAR_TEST: 1.0
+### SUBSTRAIT_INCLUDE: extensions/functions_arithmetic_decimal.yaml
+
+# basic: Basic examples without any special cases
+power(8::decimal, 2::decimal<38, 0>) = 64::fp64
```

Comment:
BFT will need some way to say "this test is disabled".  Can we add a way to reference the tests?

Reply:
I don't really understand why BFT would disable a test. A function makes sense but not a test from my pov.

If important (albeit strange), I'd suggest that BFT just list known test failures. Test cases are only a single line.

Reply:
We'd want to disable/skip a specific test if the function has multiple options.  Where different engines support different options.

Reply:
I don't understand your comment Rich. The dialect should declare what option values are allowed for which functions. The test should skip test cases whose combination is not supported by a dialect. Why would we need to refer to test cases?

Reply:
I think the issue is that if we don't declare an option, engines are free to choose _any_ option they support. For testing purposes this is tricky because we can't assume that all engines have the same defaults, and in this case we can get different answers for the same test.

Reply:
When options specified as part of the testcase, those tests are run only with the dialects where the option is supported. AFAIK currently in BFT, all tests without any options they are run on all dialects.

Reply:
@scgkiran is correct.  If a test case doesn't specify an option in the BFT, all dialects will run it.  So in this new test file format.  If the dialect doesn't support the option specified, that test case would be skipped.  BFT also skips tests if a type is unsupported (i.e. some dialects don't have decimal support)

Reply:
So I think we are ok if I read this correctly?

If no options are specified then the test should not depend on the options (should give the same answer regardless of what options are present)

If a test case's answer depends on one or more options then those options **must** be listed as part of the test.

Reply:
There should be a clear way of referring to a test so that an external system can skip a particular function's test if necessary.  That may be just <group name> + <test number within that group>.